dev/AppNotifications/AppNotificationBuilder/AppNotificationButton.cpp

```diff
@@ -50,15 +50,15 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
 
     winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationButton AppNotificationButton::SetInvokeUri(winrt::Windows::Foundation::Uri const& protocolUri)
     {
-        THROW_HR_IF(E_INVALIDARG, m_arguments.Size() > 0u);
+        THROW_HR_IF_MSG(E_INVALIDARG, m_arguments.Size() > 0u, "You cannot add a protocol activation uri after calling AddArgument.");
```

Comment:
`!m_arguments.empty()` is better than `.Size() == 0` checks

Ditto `.empty()` is preferred over `.Size() == 0`
Also other AppNotificationBuilder code uses .empty so consistency hath its virtues :-)

Same comment applies throughout

Reply:
got a PR to address this last round of feedback, here: https://github.com/microsoft/WindowsAppSDK/pull/2801

Reply:
I guess the issue is winrt::IMap doesn't have an empty function. What do you suggest here?